### PR TITLE
Transifex ui niceties

### PR DESCRIPTION
**Pull request by orangejenny**
@mkangia I noticed in a meeting that the Application dropdown in the transifex UI is a bit long and unwieldy for ICDS, so I went to switch to to a select2 and ended up making a handful of minor UI changes.

The convert translations page's alignment was bugging me:
screen shot 2019-03-05 at 7 44 08 am

Fixed:
screen shot 2019-03-05 at 7 42 55 am

Old manage app translations page:
screen shot 2019-03-05 at 7 44 18 am

New:
screen shot 2019-03-05 at 7 43 13 am

Feature flag: "Translate Application Content With Transifex"

code buddy @millerdev

### CODE REVIEW

File: corehq/apps/translations/templates/app_translations.html [hunk 1]
```diff
@@ -17,12 +17,12 @@
         </p>
     {% else %}
         <ul class="nav nav-tabs sticky-tabs" style="margin-bottom: 10px;">
-          <li><a data-toggle="tab" href="#create-trans-form" id="create-tab"><b>{% trans "Create" %}</b></a></li>
-          <li><a data-toggle="tab" href="#update-trans-form" id="update-tab"><b>{% trans "Update" %}</b></a></li>
-          <li><a data-toggle="tab" href="#push-trans-form" id="push-tab"><b>{% trans "Push" %}</b></a></li>
-          <li><a data-toggle="tab" href="#pull-trans-form" id="pull-tab"><b>{% trans "Pull" %}</b></a></li>
-          <li><a data-toggle="tab" href="#backup-trans-form" id="backup-tab"><b>{% trans "BackUp" %}</b></a></li>
-          <li><a data-toggle="tab" href="#del-trans-form" id="delete-tab"><b>{% trans "Delete" %}</b></a></li>
```
Comment (mkangia) [inline corehq/apps/translations/templates/app_translations.html]: bold didn't look nice?

File: corehq/apps/app_manager/static/app_manager/js/widgets_v4.js [hunk 2]
```diff
@@ -1,16 +1,21 @@
...
 ], function (
     $,
+    assertProperties,
     initialPageData
 ) {
-    $(".app-manager-version-dropdown").each(function () {
-        $(this).select2({
+    var initVersionDropdown = function ($select, options) {
+        options = options || {};
+        assertProperties.assertRequired(options, [], ['url', 'width']);
```
Comment (mkangia) [inline corehq/apps/app_manager/static/app_manager/js/widgets_v4.js]: Just to confirm my understanding here, this means options should be present with url and width?

if thats true wont it make the check for options.url for ajax call redundant ?

**Review by mkangia — APPROVED** [review]
Thank you!

File: corehq/apps/translations/templates/app_translations.html (hunk 1 above)
Comment (orangejenny) [reply corehq/apps/translations/templates/app_translations.html]: There needs to be a real reason ;)

File: corehq/apps/app_manager/static/app_manager/js/widgets_v4.js (hunk 2 above)
Comment (orangejenny) [reply corehq/apps/app_manager/static/app_manager/js/widgets_v4.js]: Oops, I used the wrong method. Updated to use `assert`, making it so url and width are both allowed but optional, and nothing other than url or width is allowed.

File: corehq/apps/translations/templates/app_translations.html (hunk 1 above)
Comment (mkangia) [reply corehq/apps/translations/templates/app_translations.html]: well, bold is beautiful 🅱️

**Review by millerdev — APPROVED** [review]

**Pull request merged**